input/examples/organization-elimbah-medical-center.xml

```diff
@@ -6,7 +6,7 @@
         <valueString value="Organization - Elimbah Medical Center"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="Elimbah Medical Center"/>
+        <valueMarkdown value="This is an organization example using the *AU Core Organization* profile."/>
     </extension>
```

Comment:
For all AU Core profiles don't say "using the ..." have same text as eRequesting "of ..."

Reply:
Fixed